### ISSUE TITLE
Breaking issue [Mustache 4.0.0]

### ISSUE THREAD

**Issue by 0xVavaldi** [StratumAuhuur/CTFPad#15]
## [4.0.0] / 16 January 2020
**TLDR;** if your project manipulates `Writer.prototype.parse | Writer.cache` directly or uses `.to_html()`, you probably have to change that code.

TypeError: engine.to_html is not a function
   at /root/CTFPad/node_modules/consolidate/lib/consolidate.js:1047:23
   at /root/CTFPad/node_modules/consolidate/lib/consolidate.js:161:5
   at Promise._execute (/root/CTFPad/node_modules/bluebird/js/release/debuggability.js:384:9)
   at Promise._resolveFromExecutor (/root/CTFPad/node_modules/bluebird/js/release/promise.js:518:18)
   at new Promise (/root/CTFPad/node_modules/bluebird/js/release/promise.js:103:10)
   at promisify (/root/CTFPad/node_modules/consolidate/lib/consolidate.js:154:10)
   at Function.exports.mustache.render (/root/CTFPad/node_modules/consolidate/lib/consolidate.js:1044:10)
   at /root/CTFPad/node_modules/consolidate/lib/consolidate.js:181:27
   at /root/CTFPad/node_modules/consolidate/lib/consolidate.js:101:5
   at FSReqWrap.readFileAfterClose [as oncomplete] (internal/fs/read_file_context.js:53:3)

**Comment by 0xVavaldi** [comment]
quick fix is to change package.json

    "mustache": "*",
to

    "mustache": "3.x",
